RTCPAds.js

```diff
@@ -100,18 +101,26 @@ export class RTCPAdsCarousel extends Component {
 }
 
 export class RTCPAdImageBase extends Component {
-    async openUrl(url) {
+    openUrl(url) {
         try {
-            await Linking.openURL(url);
-        } catch (error) {}
+            Linking.canOpenURL(url).then(
```

Comment:
I think we should either use Promises everywhere or use async/await everywhere.
I'm more into the newer async/await approach, which would look like below then. What do you think?
```suggestion
async openUrl(url) {
    try {
        const can = await Linking.canOpenURL(url);
        if (can) {
            await Linking.openURL(url);
        } else {
            RTCP.log(`AdImage openUrl error: Cannot open URL ${url}`);
        }
    } catch (error) {
        RTCP.log('AdImage openUrl error: ' + error.message);
    }
}
```

Reply:
agree

Reply:
resolved